engine/schema/src/com/cloud/host/dao/HostDaoImpl.java

```diff
@@ -79,6 +78,11 @@
 
     private static final String LIST_CLUSTERID_FOR_HOST_TAG = "select distinct cluster_id from host join host_tags on host.id = host_tags.host_id and host_tags.tag = ?";
 
+    protected final ConfigKey<Integer> PingInterval = new ConfigKey<Integer>("Advanced", Integer.class, "ping.interval", "60",
+            "Interval to send application level pings to make sure the connection is still working", false);
+    protected final ConfigKey<Float> PingTimeout = new ConfigKey<Float>("Advanced", Float.class, "ping.timeout", "2.5",
```

Comment:
There seems to be multiple instances/declarations of both these ConfigKeys, can you refactor them to a class/interface and consume using that. I see one in AgentManagerImpl and another in VirtualMachinePowerStateSyncImpl, and you've added one more. Please refactor, thanks.